src/Datadog.Trace.ClrProfiler.Managed/InterceptMethodAttribute.cs

```diff
@@ -42,7 +42,7 @@ public class InterceptMethodAttribute : Attribute
         /// </summary>
         public string TargetAssembly
         {
-            get => throw new NotSupportedException("Use property TargetAssemblies instead of TargetAssembly.");
+            get => string.Empty;
```

Comment:
I made this change because, while trying to debug, in Visual Studio, the `IntegrationSignatureTests`, this property getter was hit and interfered with my ability to complete the debugging.

Reply:
Consider removing the getter entirely if it's never used.

Reply:
It cannot be removed. Compiler error:  ```Error	CS0617	'TargetAssembly' is not a valid named attribute argument. Named attribute arguments must be fields which are not readonly, static, or const, or read-write properties which are public and not static.	```

Reply:
Ok, thanks for checking!